src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costs = 40;
+
+  if (days <= 0) {
+    return 'error text';
+  }
+
+  if (days < 3) {
+    return days * costs;
+  } else if (days >= 3 && days < 7) {
+    return days * costs - 20;
+  } else if (days >= 7) {
+    return days * costs - 50;
```

Comment:
Remove the unnecessary condition 'days >= 7' since it's already checked in the previous if statement.